scripts/db_restore.sh

```diff
@@ -48,40 +48,40 @@ if [ -z "$LOCALDBNAME" ]; then usage; fi;
 # where pg_stat_activity.datname = '$LOCALDBNAME'
 # EOF
 
-dropdb --if-exists $LOCALDBNAME;
-createdb $LOCALDBNAME 2> /dev/null
+dropdb -h localhost -U postgres --if-exists $LOCALDBNAME;
```

Comment:
Why is `-h localhost -U postgres` needed for every command here and afterwards? What would be the alternative?

Reply:
Thanks for leading me to a much better solution!